packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -116,6 +124,13 @@ export async function createApp() {
       'export/:surveyResponseId',
       handleWith(ExportSurveyResponseRoute),
     )
+
+    // Sync routes
+    .post<SyncStartSessionRequest>('sync', handleWith(SyncStartSessionRoute))
+    .post<SyncInitiatePullRequest>('sync/:sessionId/pull', handleWith(SyncInitiatePullRoute))
+    .get<SyncPullRequest>('sync/:sessionId/pull', handleWith(SyncPullRoute))
+    .delete<SyncEndSessionRequest>('sync/:sessionId', handleWith(SyncEndSessionRoute))
```

Comment:
We might need to add some kind of auth to these routes I think. Currently the `SessionSwitchingAuthHandler` will ferry on unauthenticated requests to their destination with a basic auth header for it's API_CLIENT user.

There's a few options we could consider:
1. Actually have a `datatrak-sync` user and login as part of the sync flow and rely on the standard session logic
2. Have a custom `SYNC_CLIENT_SECRET` or something that we validate directly in these routes

I feel like I'd maybe lean towards 1. as long as it's fine to have every client using 1 user (maybe that's not fine :thinking:)